asf_tools/water_map.py

```diff
@@ -205,9 +208,20 @@ def make_water_map(out_raster: Union[str, Path], vv_raster: Union[str, Path], vh
         raise ValueError(f'tile_shape {tile_shape} requires even values.')
 
     info = gdal.Info(str(vh_raster), format='json')
+
     out_tranform = info['geoTransform']
     out_epsg = get_epsg_code(info)
 
+    if hand_raster is None:
+        hand_raster = str(out_raster).replace('.tif', '_hand.tif')
+        log.info(f'Extracting HAND data to: {hand_raster}')
+        hand_geometry = shape(info['wgs84Extent'])
+        hand_bounds = [*info['cornerCoordinates']['upperLeft'], *info['cornerCoordinates']['lowerRight']]
+        with NamedTemporaryFile(suffix='.vrt', delete=False) as hand_vrt:
+            prepare_hand_vrt(hand_vrt.name, hand_geometry)
+            gdal.Warp(hand_raster, hand_vrt.name, dstSRS=f'EPSG:{out_epsg}',
+                      outputBounds=hand_bounds, width=info['size'][0], height=info['size'][1])
```

Comment:
This should all be a function, but I wasn't sure what to call it or where to put it...

Reply:
`prepare_hand_geotiff` or `prepare_hand_raster` would match `hyp3_gamma.dem.prepare_dem_geotiff`